lib/reunion.rb

```diff
@@ -0,0 +1,67 @@
+class Reunion
+  attr_reader :name, :activities
+  def initialize(name)
+    @name = name
+    @activities = []
+  end
+
+  def add_activity(activity)
+    @activities << activity
+  end
+
+  def total_cost
+    activities.reduce(0) do |total, activity|
+      total += activity.total_cost
+    end
+  end
+
+  def breakout
+    activities.reduce(Hash.new(0)) do |breakout, activity|
+      activity.owed.each do |participant, owed|
+        breakout[participant] += owed
+      end
+      breakout
+    end
+  end
+
+  def summary
+    breakout.map.with_index do |(participant, owed), index|
+      summary = "#{participant}: #{owed}"
+      summary += "\n" unless index == breakout.length - 1
```

Comment:
Good job! Another approach would be to just add the newline on the end of the breakout and remove it with .chomp, or you could join on a newline at the end